src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,270 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
```

Comment:
Changesets should have meaningful names (they don't have to be numeric). E.g., `id: "create ID sequence"`